src/main/java/com/locat/api/global/auth/jwt/JwtTokenProvider.java

```diff
@@ -0,0 +1,165 @@
+package com.locat.api.global.auth.jwt;
+
+import com.locat.api.global.exception.ApiExceptionType;
+import com.locat.api.global.exception.LocatApiException;
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.security.Keys;
+import jakarta.annotation.PostConstruct;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import javax.crypto.SecretKey;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.locat.api.global.constant.AuthConstant.BEARER_PREFIX;
+import static com.locat.api.global.exception.ApiExceptionType.INVALID_TOKEN;
+import static com.locat.api.global.exception.ApiExceptionType.JWT_TOKEN_INVALID;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@Component
+@RequiredArgsConstructor
+public class JwtTokenProvider {
+
+    @Value("${auth.jwt.secret}")
+    private String jwtSecret;
+    private SecretKey secretKey;
+    private final long ACCESS_TOKEN_EXPIRE_TIME = 30 * 60 * 1000L;
+    private final long REFRESH_TOKEN_EXPIRE_TIME =  7 * 24 * 60 * 60 * 1000L;
```

Comment:
`java.time` 패키지의 `Duration`을 활용해보면 어떨까요?

사람이 알아보기에도 쉽고, 시간 연산 / 단위 변환 메서드도 제공해 더 좋을 것 같아요 !

```suggestion
    private final Duration ACCESS_TOKEN_EXPIRE_TIME = Duration.ofMinutes(30);
    private final Duration REFRESH_TOKEN_EXPIRE_TIME =  Duration.ofDays(7);
```